backend/authentication/adminPasswordConfirm.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const AWS = require('aws-sdk');
+const db = new AWS.DynamoDB.DocumentClient();
+const uuid = require('uuid');
+require('dotenv').config();
+
+async function adminPasswordConfirm(event, context, callback) {
+    try {
+        // Get the recovery code from db for this user
+        const recoveryCodes = await db.scan({
+            TableName: process.env.RECOVERY_CODES_TABLE
+        }).promise();
+
+        let recoveryEntry;
+        for (let i = 0; i < recoveryCodes.Items.length; i++) {
+            if (recoveryCodes.Items[i].code == event.arguments.confirmationCode) {
+                recoveryEntry = recoveryCodes.Items[i];
+            }
+        }
+        // Check if recovery entry exists in db
+        if (!recoveryEntry) {
+            let error = new Error;
+            error.message = 'User not found!!!';
```

Comment:
This error message is lying. You're looking for a recovery code, not for the user.